CITATION.rst

```diff
@@ -1,13 +1,27 @@
+Citation Guidelines
+===================
+
+|JOSS| |Zenodo|
+
+
 If you use SkyPy for work or research presented in a publication (whether
-directly, or as a dependency to another package) we recommend and encourage
+directly, or as a dependency of another package) we recommend and encourage
 the following acknowledgment:
 
   This research made use of SkyPy, a Python package for forward modeling
-  astronomical surveys (SkyPy Collaboration, 2020).
+  astronomical surveys (Amara et. al., 2021, SkyPy Collaboration, 202x).
```

Comment:
Apologies if I have missed discussions which already address these points.
- Are we happy with 'Amara et al' or would another 'SkyPy Collaboration' be the preferred style? I appreciate there are probably reasons for this I have missed!
- We could do the work for the user of including the most up to date version year/reference, which I guess would just require changing as part of each new release.

Reply:
> * Are we happy with 'Amara et al' or would another 'SkyPy Collaboration' be the preferred style? I appreciate there are probably reasons for this I have missed!

"SkyPy Collaboration" isn't listed as an author on the JOSS paper, their template wouldn't allow it. Are you suggesting that people cite it as SkyPy Collaboration 2021 regardless? Would that work with ADS etc?

> * We could do the work for the user of including the most up to date version year/reference, which I guess would just require changing as part of each new release.

But then if somebody is using an older version they shouldn't be citing the most up to date version. It might be possible that for each release the published documentation give the corresponding DOI, but only if we can reserve zenodo DOIs before me make the release. As our Zenodo admin could you investigate this?

Reply:
I am not happy with Amara et al. either, if we could do `SkyPy Collaboration` and not mess up with ADS I would go for it.

Reply:
I cannot find anything in the ADS FAQ to answer this...

Reply:
I don't think it is possible to reserve a zenodo DOI when using the github webhook, so we would have to revert to manual submissions.

Reply:
I suppose the majority of astro-ph people will use whatever citation comes out of ADS, so we should recommend that or we will only increase confusion.

Reply:
Both Planck and DES have managed to have "[X] Collaboration" as an author on ADS. It seems as though this is picked up from the author list on the arXiv submission. I have also had good experience of the "Feedback -> Submit Updates" option on ADS.

Once a DOI and ADS Bibcode is generated, it would hopefully be possible to update it in this way.

Reply:
@rrjbca 
![Screenshot 2021-09-13 at 15 42 06](https://user-images.githubusercontent.com/3586409/133104632-c4a414ef-cf5c-4d83-aafc-9e84d60e855e.png)

Reply:
Is this something we want to try?
I'd rather have SkyPy Collaboration rather than Amara et al. 2021 but:
* Is it worth delaying the release?
* Sutieng's paper is ready and they cite Amara et al 2021.

Reply:
In my opinion, releasing this PR is our single highest priority. The proposed guidelines are adequate and these side issues are unresolved and unnecessarily holding us up. Like it or not, Amara et al 2021 is the correct citation given by both ADS and JOSS. I agree that our citation guidelines should be consistent with ADS in the first instance. I'm also against modifying the ADS entry at the expense of consistency with the entry on the JOSS website. Similarly, reserving a DOI has not been tested with the GitHub webhook and putting the wrong DOI in our citation guidelines could be incredibly harmful. I would suggest trying to reserve a DOI for our next release and if we can demonstrate that it works then integrate reserved DOIs into the citation guidelines for future releases.

Reply:
I don't disagree necessarily with any of this, but I think it is not fair labeling this a "side issue" or "unnecessary". We agreed pretty early on that all publications would be authored by "SkyPy collaboration" with an alphabetical author list following. Now reneging on this merits at least a discussion, and probably a much more visible one than hidden in this PR somewhere.

Reply:
Totally agree these are decisions beyond the scope of this PR and review that should be discussed by the collaboration. But the current proposal is at least accurate and waiting for the resolution of that discussion could cost us citations. The guidelines can always be updated in the future.

Reply:
Agreed, we'd need a conversation with the collaboration and we can also merge this as it is for the sake of the v0.5 release. After all, the issue is not about the author list (which I agree it's not a secondary issue but it is not a GitHub issue either)